optaplanner-core/src/main/java/org/optaplanner/core/impl/score/director/easy/EasyScoreCalculator.java

```diff
@@ -26,9 +26,10 @@
  * An implementation must be stateless.
  *
  * @param <Solution_> the solution type, the class with the {@link PlanningSolution} annotation
+ * @param <Score_> the score type to go with the solution
  * @see EasyScoreDirector
  */
-public interface EasyScoreCalculator<Solution_> {
+public interface EasyScoreCalculator<Solution_, Score_ extends Score<Score_>> {
```

Comment:
Hold on. This is a significant change.
I think the impact on EasyScoreCalculator and IncrementScoreCalculator should indeed be in sync, but I 'd rather decide on that separately from accepting the generic InnerScoreDirector.